desc/objectives/_coils.py

```diff
@@ -395,7 +396,7 @@ def __init__(
         name="coil curvature",
     ):
         if target is None and bounds is None:
-            bounds = (0, 1)
+            bounds = (-1, 0)
```

Comment:
might want to throw a warning if the user passes in bounds like (0, max) since that wouldn't be possible i think (if a circle has curvature -1)? For people used to unsigned curvature this might be confusing. Or maybe we just flip our sign convention so that a circle has positive curvature? though that slighlty conflicts with our convention for surface curvature

Reply:
could also have a flag for whether to use signed or unsigned curvature?

Reply:
I wish our sign convention was flipped (positive = convex), but I went with the sign convention we already use for the surface curvature. I would be fine with changing them both. 

A closed curve has to be convex at some points, but in theory you could target a curve to be locally concave in certain regions. And I think there is a conceivable use case for targeting positive curvature everywhere, even though it is not possible, to try and force the coils to be straight (low curvature everywhere). 

Instead of a flag for signed/unsigned, I think the better solution would be to add a `loss_function="abs"` option. But that can be a separate PR.

Reply:
I agree the sign convention is annoying, I based it on several textbooks which define positive curvature of a surface as curving *towards* the outward normal vector to the surface. 

I don't think we necessarily have to keep the same convention for curves, since signed curvature for non-planar curves isn't even necessarily well defined, and to me it makes more sense to define it such that a circle has positive curvature.

At the very least, I think we should include a note in the docstring for the curvature objective reminding users of the sign convention.